charts/vector/docs/quickstart.md

```diff
@@ -0,0 +1,168 @@
+
+```bash
+helm repo add datadog https://helm.datadoghq.com
+helm repo add vector https://helm.vector.dev
+helm repo update
+```
+
+Clone this repository locally to have access to the quickstart values files.
+
+```bash
+git clone https://github.com/vectordotdev/helm-charts.git && \
+	cd helm-charts/charts/vector
+```
+
+### Installing
+
+Export your Datadog API key into your local environment.
+
+```bash
+export DATADOG_API_KEY="<REPLACE_ME>"
+```
+
+Then install Vector into its own namespace, providing your API key as an
+environment variable.
+
+```bash
+helm install vector vector/vector --namespace vector --create-namespace \
+	--values docs/datadog_quickstart.yaml  --set secrets.generic.datadog_api_key="${DATADOG_API_KEY}"
+```
+
+Run the next command to confirm Vector has been configured properly and can
+connect to your Datadog account.
+
+```bash
+kubectl logs statefulset/vector --namespace vector --follow
+```
+
+Running the above will tail Vector's logs and allow to confirm your API key
+is correct. In the logs you should only see INFO messages, and three instances
+of the following log (one for each configured sink):
+
+```
+INFO vector::topology::builder: Healthcheck: Passed.
+```
+
+Once Vector has been installed and configured, you can install your Datadog Agents.
+By default the Agents will forward their logs and metrics directly to Datadog's
+hosted intake, we'll create a values file to override that behavior to forward to
+your new Vector Aggregators.
+
+For Datadog Agents version `7.35`/`6.35` or greater:
+
+```bash
+cat <<-'VALUES' > dd-agent.yaml
+datadog:
+  containerExclude: "name:vector"
+  logs:
+    enabled: true
+    containerCollectAll: true
+agents:
+  useConfigMap: true
+  customAgentConfig:
+    kubelet_tls_verify: false
+    vector:
+      logs:
+        enabled: true
+        url: http://vector-haproxy.vector:8282 # Use https if SSL is enabled in Vector source configuration
+      metrics:
+        enabled: true
+        url: http://vector-haproxy.vector:8282 # Use https if SSL is enabled in Vector source configuration
+VALUES
+```
+
+For Datadog Agents version `7.34`/`6.34` and older:
+
+_NOTE: Metrics will not be routed through Vector for Datadog Agent versions older
+than `7.35`/`6.35`._
+
+```bash
+cat <<-'VALUES' > dd-agent.yaml
+datadog:
+  containerExclude: "name:vector"
+  logs:
+    enabled: true
+    containerCollectAll: true
+agents:
+  useConfigMap: true
+  customAgentConfig:
+    kubelet_tls_verify: false
+    logs_config:
+      logs_dd_url: vector-haproxy.vector:8282
+      logs_no_ssl: true # Set to false if SSL is enabled in Vector source configuration
+      use_http: true
+VALUES
+```
+
+**WARNING:** The above config disables TLS verification of the connection to the
+Kubelet. This is useful in a quickstart environment like `minikube`, but when
+real certificates are available, the `kubelet_tls_verify` option should be removed.
```

Comment:
I'm not super happy with the phrasing here, just want to _not_ recommend turning off security features in prod

Reply:
I think it might be better to format the above snippet as:

```
agents:
  useConfigMap: true
  customAgentConfig:
    # NOTE: If you're using a quickstart environment like `minikube`, include the line below; otherwise, we recommend leaving it with the default setting of `true`.
    # kubelet_tls_verify: false
```

Abstractly, I think it's good to treat things like `minikube` as deviations, and make recommendations prod-ready by default.

Reply:
Sorta feel like making prod-ready rec's in a "quickstart" guide seems backwards 🤔